docs/cody/capabilities/agentic-chat.mdx

```diff
@@ -0,0 +1,105 @@
+
+## What can Agentic Chat do?
+
+Agentic Chat can help you with the following:
+
+### Tool Usage
+
+It has access to a suite of tools for retrieving relevant context. These tools include:
+
+- **Search Tool**: Searches the codebase using keywords extracted from your query
+- **File Tool**: Retrieves the content of specific files in your codebase
+- **CLI Tool**: Executes terminal commands to gather system or project-specific information
+- **Memory Tool**: Allows the agent to create personal notes that can be used across chat sessions
+- **Web Browser Tool (via OpenCtx)**: Searches the web for live context
+- **Linear Issue Tool (via OpenCtx)**: Integrates with Linear issue tracking when the provider is configured
+
+It integrates seamlessly with external services, such as web content retrieval and issue tracking systems, using OpenCtx providers. To learn more, [read the OpenCtx docs](/cody/capabilities/openctx).
+
+Memory allows the Agentic Chat to learn and remember your preferences, enabling a more personalized experience. You can ask it to **remember** specific details for future interactions.
+
+<Callout type="info">Tool calling is not supported on the Web. It only works with VS Code and JetBrains editor extensions.</Callout>
+
+## Terminal commands
+
+Agentic Chat can use the CLI Tool to execute shell commands and gather context from your terminal. Its ability to execute terminal commands enhances its context-gathering capabilities. However, it’s essential to understand that any information accessible via your terminal could potentially be shared with the LLM. It's recommended not to request information that you don't want to share. Here's what you should consider:
+
+- **Trusted workspaces only**: Commands can only be executed within trusted workspaces with a valid shell
+- **Potential data sharing**: Any terminal-accessible information may be shared with the LLM
+- **Allow list configuration**: This feature is disabled if the allow list is empty. By default, the allow list is set to ['*'], which allows all commands. For enhanced security, use specific command prefixes instead of the wildcard *.
+
+Commands are generated by the agent/LLM based on your request. Avoid asking it to execute destructive commands if you do not want them to run.
+
+### Configure shell command execution
+
+You can configure shell command execution via the `cody.agentic.context` setting in your editor:
+
+#### Enable Agentic Chat access to CLI
+
+Update the `cody.agentic.context` setting to `allow` or `block` specific commands:
+
+```json
+{
+  "cody.agentic.context": {
+    "shell": {
+  // Array of allowed command prefixes, or ["*"] for all commands
+      "allow": ["git", "gh", "ls"],
+      "block": ["git"] // Additional commands to block
+    }
+  }
+}
+```
+
+#### Disable Agentic Chat access to CLI
+
+To disable the feature, navigate to your user settings and set `cody.agentic.context` to an empty value. When disabled, the chat agent cannot access the CLI.
```

Comment:
@MaedahBatool  we have moved the settings out of the json file into the UI: 

<img width="664" alt="image" src="https://github.com/user-attachments/assets/dbf45962-802c-4649-8446-1d2fc68c356d" />

```suggestion
```